db/scripts/seed_database.js

```diff
@@ -27,8 +27,8 @@ async function seedDatabase() {
 
   await db.result(
     `INSERT INTO scheduled_calls_table
-    (patient_name, recipient_email, recipient_name, call_time, call_id, provider, ward_id, call_password)
-    VALUES ('Alice', 'bob@example.com', 'Bob', CURRENT_TIMESTAMP, '123', 'whereby', $1, 'password')`,
+    (patient_name, recipient_email, recipient_name, call_time, call_id, provider, ward_id, call_password, status)
+    VALUES ('Alice', 'bob@example.com', 'Bob', CURRENT_TIMESTAMP + interval '1 hour', '123', 'whereby', $1, 'password', 'scheduled')`,
```

Comment:
Is it worth using the `src/helpers/visitStatus.js` here?

Reply:
I tried to do that and it didn't like the import when running the npm db:seed task. Not sure why - perhaps ES6 isn't supported in that scenario. Any ideas?

Reply:
correct this file isn't transpiled so you won't be able to pull in the constants